docs/source/architectural_design.rst

```diff
@@ -0,0 +1,54 @@
+Architectural design
+====================
+
+Overview
+--------
+
+Given a configuration file (Data Resource Schema) VIA a REST API the Data Resource Generator (DRG) will dynamically generate:
+
+#. Database tables with relationships
+    * SQLAlchemy Object Relation Mappings (ORM)
+    * Generate and run database migrations
+#. A RESTful API
+    * Running Flask application
+    * Open API Specification 3.0
+
```

Comment:
I think lines 7-14 should appear in "Getting Started," since they provide a clear, enumerated description of what the DRG actually does. "Architectural Design" has more to do with "how this thing is built" and less to do with "what this thing can do."

Reply:
Actually....the material on lines 24-27 can be moved to the "Get Started" page. I'd also suggest re-writing them:

> The DRG comes to life when a user or client sends a Data Resource Schema, embedded inside a Data Resource Generation Payload, to the DRG generation route. Upon the receipt of a Data Resource Generation Payload, the application then generates:
>
> * relevant ORM models based on your table schema document
> * relevant REST routes based on your swagger document